azure/template.json

```diff
@@ -39,6 +39,28 @@
         "configurationStorageConnectionString": {
             "type": "securestring"
         },
+        "printFunctionFlowFunctionSchedule": {
+            "type": "string",
+            "metadata": {
+                "description": "PrintFunctionFlowFunction CRON Schedule e.g. 0 */15 * * * *"
+            }
+			,
+            "defaultValue": "0 */15 * * * *"
+        },
+        "printNotificationFunctionSchedule": {
+            "type": "string",
+            "metadata": {
+                "description": "PrintNotificationFunction CRON Schedule e.g. 0 */15 * * * *"
+            },
+            "defaultValue": "0 13 * * *"
+        },
+        "deliveryNotificationFunctionFunctionSchedule": {
+            "type": "string",
+            "metadata": {
+                "description": "DeliveryNotificationFunctionFunction CRON Schedule e.g. 0 */15 * * * *"
+            },
+            "defaultValue": "0 15 * * *"
```

Comment:
is there a missing CRON increment `*` here

Reply:
Looking online  "0 15 * * *" should run at 3pm every day (only 3 asterix)